test/integration_test.coffee

```diff
@@ -0,0 +1,107 @@
+async = require 'async'
+should = require 'should'
+child_process = require 'child_process'
+redis = require 'redis'
+RateLimit = require '../src/index'
+
+describe 'RateLimit', ->
+  redisClient = null
+  redisProcess = null
+  ratelimit = null
+
+  before (done) ->
+    # Start Redis
+    redisProcess = child_process.spawn 'redis-server',
```

Comment:
Are you sure you want to handle this in the test script? Travis can managed this if you include redis as a process in your `.travis.yml` file.